Removes ActorsController and related code

awead opened the pull request:
After the 4.2.0 update, the modal-based UI for creating actors is no longer needed.

Fixes #877

awead commented on config/application.rb:
```diff
@@ -93,5 +93,8 @@ class Application < Rails::Application
     config.action_mailer.default_url_options = { host: config.service_instance, protocol: 'https' }
+
+    # Set the  system to read only mode.  Does not allow new uploads, file edits, new collections, and collection edits
+    config.read_only = false
```
@cam156 you could use our application.yml file and do `ENV.fetch("read_only", false)` that way we should be able to invoke read-only mode by just changing the yaml file on the server instead of the application code... but either way, this looks good to me.

cam156 replied:
@awead good idea I will change that.

awead commented:
@cam156 can you squash these? I was going to do it via GitHub, but I'm wary...

cam156 commented:
@awead I did the conflict resolution in github just for grins and giggles.  I will go back and do it from the command line.

awead commented:
@cam156 huh. never done that before...

awead committed:
Removes ActorsController and related code

After the 4.2.0 update, the modal-based UI for creating actors is no
longer needed.
